frontend/src/App.js

```diff
@@ -1,14 +1,27 @@
 import logo from './logo.svg';
 import './App.css';
 import LandingPage from './components/LandingPage/LandingPage';
+import SamplePage from './components/SamplePage/SamplePage';
```

Comment:
Just wanted to add a note here that we should remove the sample page before we merge.

Reply:
> Great work Sunny! No major changes to note. Just wanted to discuss moving to react16 for enzyme.

Before we downgrade to react 16, try this [unofficial adapter](https://www.npmjs.com/package/@wojtekmaj/enzyme-adapter-react-17) for react 17 for enzyme. This should help with the mount issue.